### PR TITLE
feature/mp_ozone

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
added mp_ozone_diags.py, edited plot_diags.py to change plot title and save file, and added analysis_use to ozinfo2yaml.py

### CODE REVIEW

File: pyGSI/plot_diags.py [hunk 1]
```diff
@@ -254,15 +259,17 @@ def _get_save_file(metadata):
     
     # Save file label
     if metadata['Diag File Type'] == 'conventional':
-        save_file = '{Date:%Y%m%d%H}_{Obs Type}_{Variable}_{Diag Type}_'.format(
+        save_file = '{Date:%Y%m%d%H}_{Obs Type}_{Variable}_{Diag Type}'.format(
```
Comment (CoryMartin-NOAA) [inline pyGSI/plot_diags.py:262]: are these supposed to have suffixes? If so, I don't see where it is added/what the final form of the save_file will be.

Comment (CoryMartin-NOAA) [reply pyGSI/plot_diags.py:262]: Or is this just for the label on the figure? If this, is there a better name than 'save_file'?

File: scripts/mp_ozone_diags.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import yaml
+from multiprocessing import Pool
+import sys
+from pyGSI.diags import Ozone
+from pyGSI.plot_diags import plot_spatial, plot_histogram
+from datetime import datetime
+
+start_time = datetime.now()
+
+def plotting(ozone_config):
```
Comment (CoryMartin-NOAA) [inline scripts/mp_ozone_diags.py:12]: How is this file different from the other file? mp_plot_ozone_diags.py  Is this a mistake/error in filename change?

Comment (kevindougherty-noaa) [reply scripts/mp_ozone_diags.py]: yes, I named it wrong at first adn forgot to delete the other one.

File: pyGSI/plot_diags.py (hunk 1 above)
Comment (kevindougherty-noaa) [reply pyGSI/plot_diags.py:262]: This is the saved file name. The suffixes come from the metadata dictionary.

Comment (CoryMartin-NOAA) [reply pyGSI/plot_diags.py:262]: Yeah I see now, got it!

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**